src/lib/EasyEdit.jsx

```diff
@@ -18,7 +18,8 @@ export default class EasyEdit extends React.Component {
       editing: false,
       hover: false,
       value: props.value,
-      tempValue: null
+      tempValue: null,
+      allowEdit: props.allowEdit
```

Comment:
Is is necessary do define this in `state`as well? Can;t you just refer to the prop?